assets/arc-0031/client/utils/hooks/useAuth.ts

```diff
@@ -51,15 +55,17 @@ export const useAuth = () => {
     showErrorNotification(normalizeError(error))
   }
 
+  const prefix = 'arc0031'
+
   const signIn = async () => {
     try {
       await connectWallet()
-      const message = await new Arc31ApiClient().request(address.value)
-      // Remove the 'AX' prefix and decode the message from base64
-      const decodedMessage = decode(Buffer.from(message.replace(/^AX/, ''), 'base64'))
-      // Extract the authMessage from the decoded message
-      const authMessageWrapper = decodedMessage as { 'arc31:j': AuthMessage }
-      authMessage.value = authMessageWrapper['arc31:j']
+      const chainId = env.client.algorandChainId
+      const message = await new Arc31ApiClient().request(address.value, chainId.toString())
```

Comment:
chainId is already a string
```suggestion
      const message = await new Arc31ApiClient().request(address.value, chainId)
```

Reply:
Done !